.github/workflows/create_release_assets.yml

```diff
@@ -41,7 +46,7 @@ jobs:
       - name: Rename Release (Unix)
         run: |
           cargo install default-target
-          mkdir assets
+          mkdir -p assets
```

Comment:
Why the `-p` option? The `assets` directory is under the CWD

Reply:
It is, but if the assets directory doesn't exist for some reason (say, if the steps here get moved into a different order in the future), then I wanted to create it.  However, if it _does_ already exist (such as here), then if `-p` is omitted, the `mkdir` command will fail with a non-zero exit code and fail the job entirely.

Basically, I did that for safety, and to make it easier for future maintenance for moving the different steps around in the future if needed.  If you'd rather, I can remove the `-p` option, though.

Reply:
Get it, makes sense to me:)